aggregator/src/aggregator/error.rs

```diff
@@ -26,8 +26,8 @@ pub enum Error {
     Message(#[from] janus_messages::Error),
     /// Corresponds to `reportRejected`, §3.2
     #[error("task {0}: report {1} rejected: {2}")]
-    ReportRejected(TaskId, ReportId, Time),
-    /// Corresponds to `reportTooEarly`, §3.2. A report was rejected becuase the timestamp is too
+    ReportRejected(TaskId, ReportId, Time, ReportRejectedReason),
+    /// Corresponds to `reportTooEarly`, §3.2. A report was rejected because the timestamp is too
     /// far in the future, §4.3.2.
```

Comment:
I'd be rather surprised if these section references were still valid, several DAP drafts later. I think we should delete them. We could replace them with links to the relevant text in the current DAP draft, because at least then it's clear which draft of the document is being referred to. n.b. I'm not proposing cleaning up all section references in Janus or even in this enum in this PR, but we can improve these lines since they have been touched anyway.

Reply:
For this feedback https://github.com/divviup/janus/pull/2311/commits/c46141f1846cb8ac44158cba9a3ef9579e76fde8, I cleaned up some comments to use updated URLs and drop section references.